admin/scripts/sync_issues.py

```diff
@@ -0,0 +1,199 @@
+#!/bin/python
+
+import argparse
+import logging
+import sys
+from dataclasses import dataclass, field
+from time import sleep
+from typing import List
+
+import ijson
+import pandas as pd
+from common.ghkit import GitClient
+from common.models.notion import Project, Subgrant
+from common.utils import (
+    cleanup_empty,
+    cleanup_urls,
+    dir_path,
+    get_notion_projects,
+    load_credentials,
+    remove_urls,
+    zip_file_type,
+)
+from pandas import Series
+from pydantic import BaseModel, ValidationError
+from tqdm import tqdm
+
+
+@dataclass
+class Cli:
+    def __init__(self) -> None:
+        self.parser = argparse.ArgumentParser(
+            description="""
+            Read project information exported from the NLnet dashboard and Notion, and create one GitHub issue per project.
+            """
+        )
+
+        # Positional
+        self.parser.add_argument(
+            "notion_zip",
+            type=zip_file_type,
+            help="ZIP file containing the exported data from Notion",
```

Comment:
```suggestion
            help="ZIP file containing the exported list of projects from Notion in CSV format",
```

Reply:
I don't believe we need to specify the format of the contents here as the user should only care about supplying the ZIP. For example, we might export sub-pages later on and do some processing on their Markdown files, but all of that is invisible to the user.